samples/RiteKit.gs

```diff
@@ -0,0 +1,53 @@
+/*
+ * This sample demonstrates how to configure the library for the RiteKit API.
+ * Instructions on how to generate OAuth credentuals is available here:
+ * https://ritekit.docs.apiary.io/#introduction/options-for-authorizing-api-calls
+ */
+
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+
+/**
+ * Authorizes and makes a request to the RitKit API.
+ */
+function run() {
+  var service = getService();
+  if (service.hasAccess()) {
+    console.log(service.getAccessToken());
+    var url = 'https://api.ritekit.com/v1/stats/multiple-hashtags?tags=php&access_token=' + service.getAccessToken();
```

Comment:
Optional: It looks like the API supports passing the access token in the header as well, which is what we show for most samples. Worth doing that here as well for consistency?

Reply:
I've wrote to their support about this and I'm still waiting. 

In my case I was need to use this as a module over a common system.

Let's waiting them.

I don't think that is a good example too.

2 days please.

Reply:
The docs here say that the header should work: https://ritekit.docs.apiary.io/#introduction/options-for-authorizing-api-calls (see option B).